pkg/kubectl/cmd/util/factory.go

```diff
@@ -940,16 +940,44 @@ func GetFirstPod(client coreclient.PodsGetter, namespace string, selector labels
 	return pod, 1, nil
 }
 
-// TODO: We need to filter out stuff like secrets.
-func (f *factory) Command() string {
+func (f *factory) Command(enableSecret bool) string {
```

Comment:
maybe `printSecret`, `displaySecret`, or `showSecret` instead? Is there ever a case where we'd want to display the secret? Do we need to have that option?